src/main/java/uk/gov/hmcts/divorce/common/model/CaseData.java

```diff
@@ -603,26 +603,18 @@
 
     @CCD(
         label = "Documents uploaded",
-        typeOverride = Collection,
-        typeParameterOverride = "DivorceDocument",
-        access = {DefaultAccess.class}
-    )
-    private List<ListValue<DivorceDocument>> documentsUploaded;
-
-    @CCD(
-        label = "Uploaded supporting documents",
         hint = "Such as marriage/civil union certificate, proof of name changes, certified translations etc.",
```

Comment:
The screenshots for https://tools.hmcts.net/jira/browse/NFDIV-581 do not contain this hint (shall we remove this as well?)